onnxruntime/python/tools/automl/README.md

```diff
@@ -0,0 +1,27 @@
+# DataFrameTool overview
```

Comment:
Is this generic enough such that we don't have to store it in the automl folder?

Reply:
I do not have strong opinion on that. I saw python folder and it matched to what I was doing. Strictly speaking, it is not specific to anything. I am not even importing pandas in the class file. As long as something conforms to that api.

---
In reply to: [349512057](https://github.com/microsoft/onnxruntime/pull/2456#discussion_r349512057) [](ancestors = 349512057)

Reply:
If it's generic, please consider putting this in a non-project specific folder.

Reply:
Agreed to defer to the future as this code is still evolving.

---
In reply to: [349780293](https://github.com/microsoft/onnxruntime/pull/2456#discussion_r349780293) [](ancestors = 349780293)